fuzzers/binary_only/frida_executable_libpng/Justfile

```diff
@@ -24,6 +24,7 @@ harness: lib
 [unix]
 build:
     cargo build --profile {{ PROFILE }}
+    [ ! -d ./corpus ] && cp -r {{PROJECT_DIR}}/../../../seeds/pngs ./corpus || exit 0
```

Comment:
wait, why copy this over everytime? this is super weird.
Examples shouldn't access files outside of their own folder I think